ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4760,4 +4760,234 @@ HWRWISOVPM:
+  - outside_air_damper_percentage_sensor_2
+  - outside_air_damper_percentage_command_1
+  - outside_air_damper_percentage_command_2
+  implements:
+  - CONTROL  
+  
+RAIDS3X:
+  description: "Return air isolation damper control and monitoring, where there are three separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2
+  - return_air_isolation_damper_status_3  
+  implements:
+  - MONITORING
+
+RAIDS2X:
+  description: "Return air isolation damper control and monitoring, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2 
+  implements:
+  - MONITORING
+
+RAIDS4X:
+  description: "Return air isolation damper control and monitoring, where there are four separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - return_air_isolation_damper_status_1
+  - return_air_isolation_damper_status_2
+  - return_air_isolation_damper_status_3
+  - return_air_isolation_damper_status_4  
+  implements:
+  - MONITORING
+
+SFRMM:
+  description: "Supply fan run mode monitoring."
+  is_abstract: true
+  uses:
+  - supply_fan_run_mode
+  implements:
+  - MONITORING
+
+SRC:
+  description: "Very basic run scheduling command. "
+  is_abstract: true
+  uses:
+  - schedule_run_command
+  implements:
+  - CONTROL
+
+IDSSC:
+  description: "isolation damper status monitoring on the supply side."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status
+  implements:
+  - MONITORING
+
+IDSSC2X:
+  description: "isolation damper status monitoring on the supply side, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2  
+  implements:
+  - MONITORING
+
+IDSSC3X:
+  description: "isolation damper status monitoring on the supply side, where there are three separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2
+  - supply_air_isolation_damper_status_3
+  implements:
+  - MONITORING   
+  
+SSPM2X:
+  description: "Static pressure monitoring on the supply side where the average of the two sensors are being taken."
+  is_abstract: true
+  uses:
+  - supply_air_static_pressure_sensor_1
+  - supply_air_static_pressure_sensor_2  
+  implements:
+  - MONITORING
+
+SFRDM:  
+  description: "Supply fan run duration monitoring "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CPRDM:  
+  description: "Compressor run duration monitoring "
+  is_abstract: true
+  uses:
+  - compressor_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CMPC:
```

Comment:
What does it control to? I think this should be covered by one of the many control abstract types, such as `DXZTC`, `DXRC`, `DXSC`, etc.; why would we have this separate from the things it is controlling to?

Reply:
Deleted CMPC. Replaced by DXSC